documentation/asciidoc/computers/configuration/raspi-config.adoc

```diff
@@ -1,7 +1,7 @@
 [[raspi-config]]
 == The `raspi-config` Tool
 
-`raspi-config` is the Raspberry Pi configuration tool originally written by https://github.com/asb[Alex Bradbury]. You will be shown `raspi-config` on first booting into Raspberry Pi OS. To open the configuration tool after this,a simply run the following from the command line:
+`raspi-config` is the Raspberry Pi configuration tool originally written by https://github.com/asb[Alex Bradbury]. You will be shown `raspi-config` on first booting into Raspberry Pi OS. To open the configuration tool after this, simply run the following from the command line:
```

Comment:
"You will be shown `raspi-config` on first booting into Raspberry Pi OS."

@aallan I think it _used_ to do this many moons ago (back when Raspbian booted directly to text-mode instead of GUI-mode), but I don't think it does this any more? :confused: 
IIRC the GUI version of Raspberry Pi OS _does_ display the [piwiz](https://github.com/raspberrypi-ui/piwiz) GUI on first boot, but that's distinct from the [raspi-config](https://github.com/RPi-Distro/raspi-config) CLI tool.

ping also @XECDesign

Reply:
Yup. Fixed in 3236ded.